java/test/src/main/java/test/Ice/interrupt/AllTests.java

```diff
@@ -120,25 +108,6 @@ public static void allTests(test.TestHelper helper) throws InterruptedException
             cb.check();
 
             ExecutorService executor = java.util.concurrent.Executors.newFixedThreadPool(1);
-            executor.submit(
-                    () -> {
-                        try {
-                            Thread.sleep(500);
-                        } catch (InterruptedException e) {
-                            test(false);
-                        }
-                        mainThread.interrupt();
-                    });
-            try {
-                test(!mainThread.isInterrupted());
-                p.sleep(2000);
-                test(false);
-            } catch (com.zeroc.Ice.OperationInterruptedException ex) {
-                // Expected
-            } catch (test.Ice.interrupt.Test.InterruptedException e) {
-                test(false);
-            }
-
```

Comment:
We should revisit this test in a follow-up PR. For example, the code below seems to test Java not Ice, which makes no sense.